interfaces/acados_matlab_octave/ocp_compile_interface.m

```diff
@@ -112,82 +121,41 @@ function ocp_compile_interface(opts)
     LDFLAGS = 'LDFLAGS=$LDFLAGS -fopenmp';
 end
 
-% remove flag file, if present, before creating a new one
-flag_files = dir(fullfile(opts.output_dir,'_compiled*'));
-if ~isempty(flag_files)
-    delete(fullfile(opts.output_dir,flag_files(1).name));
-end
-% is qpOASES?
-with_qp_qpoases = ~isempty(strfind(opts.qp_solver, 'qpoases'));
-if with_qp_qpoases
-    % flag file to remember if compiled with qpOASES
-    flag_file = fullfile(opts.output_dir, '_compiled_with_qpoases.txt');
-    flagID = fopen(flag_file, 'w');
-    fclose(flagID);
-    % additional compilation flag
-    FLAGS = [FLAGS, ' -DACADOS_WITH_QPOASES'];
-end
-% is OSQP?
-with_qp_osqp = ~isempty(strfind(opts.qp_solver, 'osqp'));
-if with_qp_osqp
-    % flag file to remember if compiled with OSQP
-    flag_file = fullfile(opts.output_dir, '_compiled_with_osqp.txt');
-    flagID = fopen(flag_file, 'w');
-    fclose(flagID);
-    % additional compilation flag
-    FLAGS = [FLAGS, ' -DACADOS_WITH_OSQP'];
-end
-% is qpDUNES?
-with_qp_qpdunes = ~isempty(strfind(opts.qp_solver, 'qpdunes'));
-if with_qp_qpdunes
-    % flag file to remember if compiled with qpDUNES
-    flag_file = fullfile(opts.output_dir, '_compiled_with_qpdunes.txt');
-    flagID = fopen(flag_file, 'w');
-    fclose(flagID);
-end
-% is HPMPC?
-with_qp_hpmpc = ~isempty(strfind(opts.qp_solver, 'hpmpc'));
-if with_qp_hpmpc
-    % flag file to remember if compiled with HPMPC
-    flag_file = fullfile(opts.output_dir, '_compiled_with_hpmpc.txt');
-    flagID = fopen(flag_file, 'w');
-    fclose(flagID);
-    % additional compilation flag
-    FLAGS = [FLAGS, ' -DACADOS_WITH_HPMPC'];
+if ~is_octave()
```

Comment:
@asparc your PR didn't include these flags as far as I see, which is weird because I needed them.

Reply:
oops, right..